templates/etc/httpd/conf.d/05-foreman.d/katello.conf.erb

```diff
@@ -6,7 +6,9 @@
 Alias /pub /var/www/html/pub
 
 <Location /pub>
-  PassengerEnabled off 
+<% if @use_passenger %>
```

Comment:
I think this should be `scope['::katello::use_passenger']`, as it's defined in katello class and called from katello::config